File: php/swap.php
<?php

$x = "a";
$y = 1237;

echo "Before swap:\n";
echo "X: {$x} - Y: {$y}\n";

[$x,$y] = [$y,$x];

echo "After swap:\n";
echo "X: {$x} - Y: {$y}\n";

?>

File: c#/swap.cs
using System;

namespace Swap
{
    class MainClass
    {
        public static void Main(string[] args)
        {
            int a = 10;
            int b = 2;

            (a, b) = (b, a);

            Console.WriteLine(a);
            Console.WriteLine(b);

        }
    }
}